ConnectionFactory.php

```diff
@@ -132,6 +132,8 @@ public function createConnection(array $params, ?Configuration $config = null, ?
      * and the platform version is unknown.
      * For details have a look at DoctrineBundle issue #673.
```

Comment:
@ostrolucky 

The reference is already here, just not clickable.

Reply:
Hmm I would say perhaps better then to change it to a link rather than adding a link additionally to ID